portfolio/src/main/webapp/script.js

```diff
@@ -99,13 +101,25 @@ function createIndividualComment(comment) {
 function addUploadedImageToComment(comment) {
   const imageElement = document.createElement('img');
   imageElement.setAttribute('class', 'comment-image');
-  imageElement.src = comment.imageURL;
+  imageElement.src = `/serve?blob-key=${comment.blobKeyString}`;
   const imageAnchor = document.createElement('a');
-  imageAnchor.href = comment.imageURL;
+  imageAnchor.href = `/serve?blob-key=${comment.blobKeyString}`;
   imageAnchor.appendChild(imageElement);
   return imageAnchor;
 }
 
+function addImageLabels(labels) {
```

Comment:
This function name is confusing, as it only creates the image labels without adding them anywhere. It's the caller which does the adding.

Same for addUploadedImageToComment

Reply:
done